sys/drv/pit.c

```diff
@@ -14,43 +14,67 @@ typedef struct pit_state {
   spin_t lock;
   resource_t *irq_res;
   timer_t timer;
-  uint64_t period_frac;        /* period as a fraction of a second */
-  uint16_t period_cntr;        /* period as PIT counter value */
-  volatile uint16_t last_cntr; /* last seen counter value */
-  volatile bool overflow;      /* counter overflow detected? */
-  bintime_t time;              /* last time measured by the timer */
+  uint16_t period_cntr;     /* period as PIT counter value */
+  uint16_t counter16_last;  /* last seen counter value */
```

Comment:
Could you change `counter16_last` and `counter64_last` into somewhat more self-descriptive identifiers?

Reply:
Done.